src/pixi.inspector.js

```diff
@@ -225,7 +225,14 @@
 			var formatted = {
 				_inspector: window.$pixi._inspector
 			};
-			Object.keys(window.$pixi).forEach(function (property) {
+
+			var properties = Object.keys(window.$pixi);
+
+			if (formatted._inspector !== undefined && formatted._inspector.whitelist !== undefined) {
+				properties = formatted._inspector.whitelist;
+			}
+
+			properties.forEach(function (property) {
```

Comment:
By default, display the full list of properties. If a whitelist has been specified on `_inspector`, use the whitelisted properties.